index.html

```diff
@@ -1015,7 +1015,8 @@ <h2 id="arrays">Array Functions</h2>
         <b class="header">object</b><code>_.object(list, [values])</code>
         <br />
         Converts arrays into objects. Pass either a single list of
-        <tt>[key, value]</tt> pairs, or a list of keys, and a list of values.
+        <tt>[key, value]</tt> pairs, or a list of keys, and a list of values. The former
+        usage is invertible through <code>_.pairs</code>.
```

Comment:
I'm on-board with this change, but I'm not sure to what "The former usage" refers. Could you reword?